frontend/src/pages/api/isadmin.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
```

Comment:
Hva er egt poenget med isadmin-routen? Både backend og frontend for så vidt

Reply:
Ja, den skulle jeg egentlig ha fjernet i https://github.com/echo-webkom/echo-web/pull/287/commits/b140e6ba1acde9d58eb0fd944471c09687bb821c